inject-groovy/src/test/groovy/io/micronaut/aop/simple/AnotherClass.groovy

```diff
@@ -29,11 +29,6 @@ class AnotherClass<A> {
         return "Name is " + name
     }
 
-    // protected methods not proxied
```

Comment:
Can you revert this? This method should not cause a problem

Reply:
Reverted.